### DISCUSSION TITLE
My ID card has the date written on the back.

### DISCUSSION THREAD

**Discussion by monoonn** [community/community#57056] [GitHub Education]
### Hi everyone,

My ID card has the date written on the back.
It is difficult because I can only send one picture to the student pack application.

**Comment by arpittyagi102** [comment]
You should try adding both the pictures into on like a collage and then send it

**Comment by samus-aran** [comment]
Hi @monoonn 👋 

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Also check for spaces between names.  It’s these small but important points to consider when applying that can make a much smoother process.

Hope this helps 👍

**Comment by Ctrasuss** [comment]
That sounds tricky with the date being on the back of your ID card. If you can only send one picture, maybe try scanning both sides and combining them into one image? This way, you can show all the necessary details in a single file.
In case you're still hunting for this info, I'd suggest looking for an identity validation service. They can help ensure everything is legit and you'll be thankful for [having the real deal](https://www.idanalyzer.com/products/id-verification-api.html). These services often have up-to-date info and can guide you on the best way to submit your ID.
Have you tried contacting the application support team for any tips on how to handle this? They might have a workaround or a preferred method for situations like this.
